src/epic-auth/backend-auth.service.ts

```diff
@@ -11,7 +11,7 @@ export class BackendServiceAuthService {
 
   constructor(private readonly jwtService: JwtService) {
     // Load private key from a file
-    this.privateKey = fs.readFileSync('private_key.pem', 'utf8');
```

Comment:
Revert this change before pushing.. i am deleting the whole epic-auth module as it is no longer needed